sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -4838,9 +4868,16 @@ pi_result piEventGetInfo(pi_event Event, pi_event_info ParamName,
       // Only do the execute of the open command list if the event that
       // is being queried and event that is to be signalled by something
       // currently in that open command list.
-      if (Event->Queue->hasOpenCommandList() &&
-          Event->Queue->OpenCommandList->first == Event->ZeCommandList) {
-        if (auto Res = Event->Queue->executeOpenCommandList())
+      if (Event->Queue->hasOpenCommandList(false /* IsCopy */) &&
+          Event->Queue->ComputeCommandBatch.OpenCommandList->first ==
+              Event->ZeCommandList) {
+        if (auto Res = Event->Queue->executeOpenCommandList(false /* IsCopy */))
```

Comment:
maybe 

```using IsCopy = bool;```

and then you can write arguments like

```IsCopy{false}```

Reply:
Done. Thanks